app/src/main/res/layout/activity_edit_modify_quiz_settings.xml

```diff
@@ -21,6 +21,19 @@
         app:layout_constraintBottom_toTopOf="@+id/button_start_editing"
         android:inputType="text" />
 
+    <CheckBox
```

Comment:
How do you handle the case when a non-treasure hunt is converted to a treasure hunt quiz ? It means that the user must add a location to all the questions. I think that the easiest thing for now is to not allow the user to change that setting.